tests/cypress/tests/unit/single_column_syntactic_tree_test.ts

```diff
@@ -97,6 +97,14 @@ describe('Single Column Syntax Tree', () => {
         expect(tree.toQueryString()).to.equal('{a} = 1');
     });
 
+    it.only('can be permissive value expression', () => {
+        const tree = new SingleColumnSyntaxTree('Hello world', COLUMN_TEXT);
+
+        expect(tree.isValid).to.equal(true);
+        expect(tree.evaluate({ a: 'Hello world' })).to.equal(true);
+        expect(tree.evaluate({ a: 'Helloworld' })).to.equal(false);
+    });
```

Comment:
Can you include a `toQueryString` test for permissive as well?

Reply:
Sure